middlewares/validators/user.js

```diff
@@ -192,10 +192,26 @@ async function validateUserQueryParams(req, res, next) {
   }
 }
 
+async function validateUpdateRoles(req, res, next) {
+  const schema = joi.object().strict().min(1).keys({
+    member: joi.boolean(),
+    archived: joi.boolean(),
+  });
```

Comment:
you can add max 1 also so only 1 key is passed. as both are not allowed at once Right ?

Reply:
Done please check my latest commit